bot/exts/recruitment/talentpool/_review.py

```diff
@@ -121,45 +119,46 @@ async def get_user_for_review(self) -> Optional[int]:
         """
         now = datetime.now(timezone.utc)
 
-        possible = []
-        for user_id, user_data in self._pool.cache.items():
-            time_since_nomination = now - isoparse(user_data["inserted_at"])
+        possible: list[Nomination] = []
+        nominations = await self.api.get_nominations(active=True)
+        for nomination in nominations:
+            time_since_nomination = now - nomination.inserted_at
             if (
-                not user_data["reviewed"]
+                not nomination.reviewed
                 and time_since_nomination > MIN_NOMINATION_TIME
             ):
-                possible.append((user_id, user_data))
+                possible.append(nomination)
 
         if not possible:
             log.debug("No users ready to review.")
             return None
 
-        oldest_date = min(isoparse(x[1]["inserted_at"]) for x in possible)
-        max_entries = max(len(x[1]["entries"]) for x in possible)
+        oldest_date = min(nom.inserted_at for nom in possible)
+        max_entries = max(len(nom.entries) for nom in possible)
```

Comment:
Suggestion: could `possible_nominations` and `possible_nomination` be a better fit as names ?